Base/usr/share/man/man1/Profiler.md

```diff
@@ -50,4 +50,3 @@ $ Profiler perfcore.123
 ## See also
 
 * [`perfcore`(5)](help://man/5/perfcore)
-
```

Comment:
Why did you remove this trailing newline as well as the others?

Reply:
I tried not to with both of the editors I use, I'm not sure. I'll try with a basic text editor maybe.